File: 001_class.py
class Person:
    def __init__(self, name, age, gender):
        self.name = name
        self.age = age
        self.gender = gender

    def introduce(self):
        print(f"Hi, I'm {self.name}, {self.age} years old, and I am {self.gender}.")

# Creating objects (instances) of the Person class
person1 = Person("Alice", 30, "female")
person2 = Person("Bob", 25, "male")

# Accessing attributes and calling methods
print(person1.name)  # Output: Alice
person2.introduce()  # Output: Hi, I'm Bob, 25 years old, and I am male.

File: 001_class.java
class Person {
    String name;
    int age;
    String gender;

    public Person(String name, int age, String gender) {
        this.name = name;
        this.age = age;
        this.gender = gender;
    }

    public void introduce() {
        System.out.println("Hi, I'm " + name + ", " + age + " years old, and I am " + gender + ".");
    }
}

public class Main {
    public static void main(String[] args) {
        Person person1 = new Person("Alice", 30, "female");
        Person person2 = new Person("Bob", 25, "male");

        System.out.println(person1.name); // Output: Alice
        person2.introduce(); // Output: Hi, I'm Bob, 25 years old, and I am male.
    }
}
